### PR TITLE
DOC: Integrate documentation for intelmq-api

### PR DESCRIPTION
**Pull request by schacht-certat**

### CODE REVIEW

File: docs/user/intelmq-api.rst [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+Intelmq Api
+###########
+
+intelmq-api is a `hug <http://hug.rest>`_ based API for the `intelmq <https://github.com/certtools/intelmq/>`_ project.
+
+**********************************
+Installing and running intelmq-api
+**********************************
+
+``intelmq-api`` requires at least `Python 3.6 <https://www.python.org/downloads/release/python-360/>`_.
```
Comment (wagner-certat) [inline docs/user/intelmq-api.rst:15]: That's the same requirement as for IntelMQ itself. Do we need to add it here explicitly again?

File: docs/user/intelmq-api.rst [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+Install ``intelmq-api`` using your preferred package installation mechanism (``pip``, ``apt``, ``yum``, ``dnf``...).
+
+* ``pip install intelmq-api``
+* ``apt install python3-intelmq-api``
+* ``yum install intelmq-api``
+* ``zypper install intelmq-api``
```
Comment (wagner-certat) [inline docs/user/intelmq-api.rst:22]: That's only possible if the intelmq repo is used.

File: docs/user/intelmq-api.rst [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+``intelmq-api`` is configured using a configuration file in ``json`` format.
+The path to the configuration file is set using the environment variable ``INTELMQ_API_CONFIG``.
+When running the API using ``hug``, you can set the environment variable like this:
+
+.. code-block:: bash
+
+   INTELMQ_API_CONFIG=/etc/intelmq-api/config.json hug -m intelmq_api.serve
+
+
+The configuration file ``/etc/intelmq-api/config.json`` which is shipped with the packages is also listed here fore reference.
```
Comment (wagner-certat) [inline docs/user/intelmq-api.rst:64]: typo "fore"

File: docs/user/intelmq-api.rst [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+If the command is not configured correctly, you'll see exceptions on startup like this:
+
+.. code-block:: bash
+
+   intelmq_manager.runctl.IntelMQCtlError: <ERROR_MESSAGE>
+
+This means the intelmqctl command could not be executed as a subprocess.
+The ``<ERROR_MESSAGE>`` should indicate why.
+
+To save the positions of the bots in the configuration map, you need an existing writable ``manager/positions.conf`` file. If it's missing, just create an empty one.
```
Comment (wagner-certat) [inline docs/user/intelmq-api.rst:112]: ... and set the owner?

File: docs/user/intelmq-api.rst [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+``intelmq-api`` requires at least `Python 3.6 <https://www.python.org/downloads/release/python-360/>`_.
+
+Install ``intelmq-api`` using your preferred package installation mechanism (``pip``, ``apt``, ``yum``, ``dnf``...).
+
+* ``pip install intelmq-api``
+* ``apt install python3-intelmq-api``
+* ``yum install intelmq-api``
+* ``zypper install intelmq-api``
+
+The ``intelmq-api`` packages ship a configuration file in ``/etc/intelmq-api/config.json`` and a virtualhost configuration file for Apache 2 in ``/etc/intelmq-api/apache-virtualhost.conf``.
```
Comment (wagner-certat) [inline docs/user/intelmq-api.rst:24]: That file needs to be adapted with the correct path. E.g. for an installation via pip on buster, the path to intelmq-api is `/usr/local/lib/python3.7/dist-packages/intelmq_api/`.

**Review by wagner-certat — CHANGES_REQUESTED** [review]

File: docs/user/intelmq-api.rst (hunk 1 above)
Comment (schacht-certat) [reply docs/user/intelmq-api.rst]: No, you're right - I replaced it and mentioned the dependency on intelmq

File: docs/user/intelmq-api.rst (hunk 2 above)
Comment (schacht-certat) [reply docs/user/intelmq-api.rst]: ACK, I've removed the list of installation commands and added a paragraph about the repository.

File: docs/user/intelmq-api.rst (hunk 3 above)
Comment (schacht-certat) [reply docs/user/intelmq-api.rst]: fixe.

File: docs/user/intelmq-api.rst (hunk 5 above)
Comment (schacht-certat) [reply docs/user/intelmq-api.rst]: Ah, damn- I've updated the paragraph and tried to explain it, I hope that covers most cases

File: docs/user/intelmq-api.rst (hunk 4 above)
Comment (schacht-certat) [reply docs/user/intelmq-api.rst]: I've now replaced that sentence with a paragraph about permissions in the configuration section.

**Review by wagner-certat — APPROVED** [review]

**Pull request merged**